File: litige-front/src/app/models/litige.ts
import { User } from './user';
import { FileInformation } from './file-information';

export class Litige {
  id: number;
  creation: Date;
  objet: string;
  message: string;
  agent: User;
  requerant: User;
  fileInformations: Array<FileInformation>;
}

File: litige-resource-1/src/main/java/com/ruscassie/litige/dto/Litige.java
package com.ruscassie.litige.dto;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class Litige {

	private Long id;
	private String objet;
	private String message;
	private Date creation;
	private User requerant;
	private User agent;

}
